kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkManagerBase.java

```diff
@@ -80,8 +80,7 @@ private static ListeningExecutorService queryThreadPool() {
    * We will not aggregate locally for future use-cases that have complex group by etc
    */
   public SearchResult<T> query(SearchQuery query, Duration queryTimeout) {
-    SearchResult<T> errorResult =
-        new SearchResult<>(new ArrayList<>(), 0, 0, new ArrayList<>(), 0, 0, 1, 0);
+    SearchResult<T> errorResult = new SearchResult<>(new ArrayList<>(), 0, 0, 0, 0, 1, 0, null);
```

Comment:
can this also be s static final ?

Reply:
It could be final, but it can't be static or moved out of this method due to it using generics.